include/Tools/Radiation/Models/Cone/ConeBremsstrahlungScreenedEmission.h

```diff
@@ -0,0 +1,72 @@
+#ifndef _CONE_BREMSSTRAHLUNG_SCREENED_EMISSION_H
+#define _CONE_BREMSSTRAHLUNG_SCREENED_EMISSION_H
+
+#include <softlib/constants.h>
+#include "Tools/Radiation/Models/Cone.h"
+#include "Tools/Radiation/Models/Cone/ConeEmission.h"
+#include "Tools/Radiation/Radiation.h"
+#include "Tools/Radiation/RadiationParticle.h"
+
+#include <gsl/gsl_integration.h>
+
+namespace __Radiation {
+    class ConeBremsstrahlungScreenedEmission : public ConeEmission {
+        private:
+#define _c  LIGHTSPEED
+#define _c2 (_c*_c)
+#define _e  ELEMENTARY_CHARGE
+#define _e2 (_e*_e)
+#define _m  ELECTRON_MASS
+#define _m2 (_m*_m)
+            static constexpr slibreal_t alpha =
+                _e*_e / (4.0*M_PI*EPS0*_c*HBAR);
+            static constexpr slibreal_t r02Alpha = 
+                _e2*alpha / (16.0*M_PI*M_PI*EPS0*EPS0*_m2*_c2);
+            unsigned int nspecies;
+            slibreal_t *Z;
+            slibreal_t *Z0;
+            slibreal_t *density;
+            slibreal_t r02;
+
+            double qagsEpsAbs=0.0,
+                   qagsEpsRel=1e-3;
+            std::size_t qagsLimit = 100;
+            gsl_integration_workspace *qagsWS = nullptr;
+
+       public:
+            ConeBremsstrahlungScreenedEmission(Detector *det, MagneticField2D *mf, unsigned int nspecies, slibreal_t *Z, slibreal_t *Z0, slibreal_t *density);
+            ~ConeBremsstrahlungScreenedEmission() {delete []this->Z; delete []this->Z0; delete []this->density;
```

Comment:
This non-trivial destructor should be placed in ``ConeBremsstrahlungScreenedEmission.cpp`` instead. Here, just declare it:
```c++
~ConeBremsstrahlungScreenedEmission();
```